### ISSUE TITLE
app_icon is not showing

### ISSUE THREAD

**Issue by aldwnesx** [MaikuB/flutter_local_notifications#631]
if ii use the flutter app_icon.png provided in your example project, it works.

If i replace it with my own and put the same name it doesn't show up...

do I have to be using certain dimensions?
I tried using @mipmap/ic_launcher as well but doesn't show up..

**Comment by MaikuB** [comment]
I don't know off the top of my head if a mismatch of dimensions will cause issues with it appearing but yes there are certain dimensions and I believe requirements on opacity etc. There are links in the readme that point to official Android documentation about notification icons.

**Comment by vadrian89** [comment]
Hi, I also hit this issue and I solved it after looking at these links:
https://documentation.onesignal.com/docs/customize-notification-icons
https://developer.huawei.com/consumer/en/doc/development/HMS-Guides/push-icon-spec

Basically I redid the icon, made multiple icons with the following size/location specifications:
res/drawable-mdpi/ (24x24)
res/drawable-hdpi/ (36x36)
res/drawable-xhdpi/ (48x48)
res/drawable-xxhdpi/ (72x72)
res/drawable-xxxhdpi/ (96x96)

Aswell I made sure the icon had a transparent background.
The page from Huawei says the following:
"The icon does not have a background. The outline of the icon follows the shape of the app logo. The area outside the outline is transparent."
